src/libcommon/utility/utility.cpp

```diff
@@ -664,6 +664,26 @@ bool CommonUtility::fileNameIsValid(const SyncName &name) {
     return true;
 }
 
+std::string CommonUtility::envVarValue(const std::string &name) {
+#ifdef _WIN32
+    char *value = nullptr;
+    size_t sz = 0;
+    if (_dupenv_s(&value, &sz, name.c_str()) == 0 && value != nullptr) {
+        std::string valueStr(value);
+        free(value);
+        return valueStr;
+    }
```

Comment:
Sonar is suggesting to avoid usage of free. What about :
`
if (size_t sz = 0; getenv_s(&sz, nullptr, 0, name.c_str()) == 0 && sz != 0) {
    auto value = std::make_unique<char[]>(sz);
    if (getenv_s(&sz, value.get(), sz, name.c_str()) == 0) {
        return std::string(value.get());
    }
}
`